org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -225,44 +229,44 @@ public void cleanQueue(String queueName) throws EventException {
 									continue;
 								}
 							}
-	
+
 							if (qbean.getStatus().isFinal()) {
 								final long submitted = qbean.getSubmissionTime();
 								final long current   = System.currentTimeMillis();
 								if (current-submitted > getMaximumCompleteAge()) {
 									removeIds.add(t.getJMSMessageID());
 								}
 							}
-	
+
 						} catch (Exception ne) {
 							logger.warn("Message "+t.getText()+" is not legal and will be removed.", ne);
 							removeIds.add(t.getJMSMessageID());
 						}
 					}
 				}
-	
+
 				// We fail the non-started jobs now - otherwise we could
 				// actually start them late. TODO check this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)